examples/rest-stickies-csv/app.js

```diff
@@ -1,16 +1,22 @@
 // Require sensitive environment variables (oauthToken, Miro boardID)
-require("dotenv/config");
+require("dotenv").config();
 
 // Require express for server and handlebars for clientside rendering
 const express = require("express");
 const exphbs = require("express-handlebars");
 const app = express();
 
-// Require axios for http requests:
-const axios = require("axios");
+const { Miro, TagCreateRequest, StickyNoteData } = require("@mirohq/miro-node");
+
+const miro = new Miro({
+  clientId: process.env.MIRO_CLIENT_ID,
```

Comment:
nitpick: this is not needed because client will use environment variables by default if no options are provided